src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/DisplayValue.js

```diff
@@ -8,7 +8,7 @@ import displayValueStyles from './displayValue.scss';
 import type {Skin} from './types';
 
 type Props = {|
-    children: string,
+    children: any,
```

Comment:
I don't know the correct type for any kind of html element here. `React.Element` didn't work, because it needs a generic type.

Reply:
i think the correct type is `Node`, at least that is what was used in other places 🙂 
https://flow.org/en/docs/react/types/#toc-react-node

Reply:
Ah I see. It look like you've already fixed it in https://github.com/sulu/sulu/pull/5654/commits/447f9ed2a087949abfe575d7106cfdc4f3665ecb :+1:. So this seems to be resolved.